modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,153 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="proxy-certificates_{context}"]
+= Proxy certificates
+
+[discrete]
+== Purpose
+
+The `trustedCA` field of `proxy` is a reference to a ConfigMap that contains a
+user-provided trusted certificate authority (CA) bundle. This bundle is merged
+with the Red Hat Enterprise Linux CoreOS (RHCOS) trust bundle and injected into
```

Comment:
We have attributes for RHCOS and the long form that should be used:

:op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
:op-system: RHCOS